projects/Mallard/src/helpers/storage.ts

```diff
@@ -68,6 +68,8 @@ const cacheClearCache = createAsyncCache<string>('cacheClear')
 
 const validAttemptCache = createAsyncCache<number>('validAttempt-cache')
 
+const loggingQueueCache = createAsyncCache<string>('loggingQueue')
```

Comment:
Just our of curiosity, what is the max limit of AsyncStorage (max data size)?

Reply:
I beleive its as large as you need (as long as there is space on the device). Perhaps we should have some cleanup functions if logs get cluttered or unsent for some reason.

Reply:
Ah, I see. I read it somewhere it was 6MB. Looks like that limit is for android but not they are still relevant.
https://github.com/react-native-community/async-storage/issues/83